### DISCUSSION TITLE
Parameter to trust HttpExporter to a Self Signed Certificate Collector

### DISCUSSION THREAD

**Discussion by johannrenck** [open-telemetry/opentelemetry-java-instrumentation#11015] [Q&A]
Is there a parameter that can be set to allow sending data to a collector that uses a self signed certificate?

**Comment by laurit** [comment]
@johannrenck see https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure, search for certificate.
